drivers/serial/uart_nrfx_uart.c

```diff
@@ -38,6 +38,33 @@ static inline const struct uart_nrfx_config *get_dev_config(struct device *dev)
 	return dev->config->config_info;
 }
 
+#ifdef CONFIG_UART_ASYNC_API
+static struct {
+	uart_callback_t callback;
+	void *user_data;
+
+	bool rx_enabled;
+	u8_t *rx_buffer;
+	u8_t *rx_secondary_buffer;
+	size_t rx_buffer_length;
+	size_t rx_secondary_buffer_length;
+	volatile size_t rx_counter;
+	volatile size_t rx_offset;
+	size_t rx_timeout;
+	struct k_delayed_work rx_timeout_work;
+
+	bool tx_abort;
+	u8_t *tx_buffer;
+	size_t tx_buffer_length;
+	volatile size_t tx_counter;
+#if	defined(DT_NORDIC_NRF_UART_UART_0_RTS_PIN) && \
```

Comment:
IMHO, having a tab after "#if", so condition has the same indent as normal code, makes it harder to read.